components/QuestionForm/QuestionForm.tsx

```diff
@@ -143,6 +189,43 @@ function QuestionFormInner({
             }}
           />
         </section>
+        <section>
+          <h4>Question Tags</h4>
+          <Select
+            backgroundColor="blue"
+            value={
+              selectedLabels.length > 0
+                ? selectedLabels[selectedLabels.length - 1]!.id
+                : "default"
+            }
+            options={optionsObject}
+            onChange={(e) => handleLabelSelect(e.target.value)}
+          />
+        </section>
+        <section
+          style={{
+            display: "flex",
+            flexWrap: "wrap",
+            gap: "1rem",
+            padding: "1rem 0",
+          }}
+        >
+          {selectedLabels.map(function (label) {
```

Comment:
just a style consistency nit here but we use arrow functions for callbacks like this throughout the codebase

Reply:
sure I'll make a note to fix this when implementing the other changes for prepopulating and displaying.